handlers/messages_handler.go

```diff
@@ -39,19 +39,24 @@ func MessagesCreate(c *gin.Context) {
     c.Fail(500, err)
   }
 
-	var json MessageJSON
+  var json MessageJSON
   c.Bind(&json)
 
+  m, err := SendMessage(user, room, json.Body);
+  if (err != nil){
+	  c.Fail(500, err)
+  }
+
+  c.JSON(200, gin.H{"message": m})
+}
+
+func SendMessage(user *models.User, room *models.Room, body string) (*models.Message, error) {
```

Comment:
Could we cut down this function signature even more so it's just `(userId string, roomId string, body string)`? (This is kind of a nit, but I think it'd make for a clearer separation of concerns.)

Also, just to be explicit, could we rename it to `SendAndCreateMessage`?